lang/ilias_pt.lang

```diff
@@ -2748,7 +2747,7 @@ chatroom_adm#:#client_settings#:#Criar link de categoria
 chatroom_adm#:#enable_osd#:#Adicionar categoria
 chatroom_adm#:#general_settings_title#:#Definições gerais de chat
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Atualizar intervalo (segundos)
+chatroom_adm#:#osd_interval#:#Atualizar intervalo (segundos)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.